docs/proposals/backup/backup.md

```diff
@@ -0,0 +1,735 @@
+	// +nullable
+	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
+
+	// OrLabelSelectors is list of metav1.LabelSelector to filter with when adding individual objects to the backup.
+	// If multiple provided they will be joined by the OR operator.
+	// LabelSelector as well as OrLabelSelectors cannot co-exist in backup request, only one of them can be used.
+	// +optional
+	// +nullable
+	OrLabelSelectors []*metav1.LabelSelector `json:"orLabelSelectors,omitempty"`
+}
+```
+
+
+The Velero support **Global Scope Resource Filtering**: When specific ns-scope resources are designated, if their descriptions involve cluster-scope resources, 
+Velero will automatically back up these essential resources without requiring users to configure them separately. However, users can still manually configure the global scope resource filtering.
+
+
+##### Unified Restore API
+
+Below is the initial design for the Unified Restore API:
+
+```console
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,categories=kurator-dev
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="Phase of the Restore"
+
+// Restore is the schema for the Restore's API.
+type Restore struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              RestoreSpec   `json:"spec,omitempty"`
+	Status            RestoreStatus `json:"status,omitempty"`
+}
+
+type RestoreSpec struct {
+	// BackupName specifies the backup on which this restore operation is based.
+	BackupName string `json:"backupName"`
+
+	// Destination indicates the clusters where restore should be performed.
+	// +optional
+	Destination *Destination `json:"destination,omitempty"`
+
+	// Policy defines the customization rules for the restore.
+	// If null, the backup will be fully restored using default settings.
+	// +optional
+	Policy *RestorePolicy `json:"policy,omitempty"`
+}
+
+// Note: partly copied from https://github.com/vmware-tanzu/velero/pkg/apis/restore_types.go
+// RestorePolicy defines the specification for a Velero restore.
+type RestorePolicy struct {
+	// ResourceFilter is the filter for the resources to be restored.
+	// If not set, all resources from the backup will be restored.
+	// +optional
+	ResourceFilter *ResourceFilter `json:"resourceFilter,omitempty"`
+
+	// NamespaceMapping is a map of source namespace names
+	// to target namespace names to restore into.
+	// Any source namespaces not included in the map will be restored into
+	// namespaces of the same name.
+	// +optional
+	NamespaceMapping map[string]string `json:"namespaceMapping,omitempty"`
+
+	// RestoreStatus specifies which resources we should restore the status field.
+	// If nil, no objects are included.
+	// +optional
+	// +nullable
+	RestoreStatus *RestoreStatusSpec `json:"restoreStatus,omitempty"`
```

Comment:
how about reserveStatus to distinguish from `RestoreStatus` struct

Reply:
ok